src/main/webapp/map.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Map Project</title>
+  <style>
+    #map{
```

Comment:
I would move this to a separate CSS file under the CSS folder if you plan to add more styling to map :)